### ISSUE TITLE
embree@3.13.1 build fails on ARM: g++: error: unrecognized command-line option '-mavx2'

### ISSUE THREAD

**Issue by eugeneswalker** [LLNL/spack#39534]
### Steps to reproduce the issue

`embree@3.13.1` build fails on ARM using:
* spack@develop (c9ec5fb9ac565033c7db643f0b51da5dc9315ac0 from `Sat Aug 19 00:46:50 2023 +0200`)
* Ubuntu 20.04, ARM64
* GCC 11.4.0
* Docker container image `ecpe4s/ubuntu20.04-runner-arm64-gcc-11.4:2023.08.01`

This error is impeding our ability to install the following ECP DAV specs:
```
  - ecp-data-vis-sdk +adios2 +hdf5 +vtkm +zfp +paraview +cuda cuda_arch=75
  - ecp-data-vis-sdk +adios2 +hdf5 +vtkm +zfp +paraview +cuda cuda_arch=80
  - ecp-data-vis-sdk +adios2 +hdf5 +vtkm +zfp +paraview +cuda cuda_arch=90
```

<details><summary>Spack environment</summary>

```
spack:
  view: false

  concretizer:
    reuse: false
    unify: false

  packages:
    all:
      require: '%gcc@11.4.0'
      providers:
        blas: [openblas]
        mpi: [mpich]
      target: [aarch64]
      variants: +mpi
    binutils:
      variants: +ld +gold +headers +libiberty ~nls
    elfutils:
      variants: +bzip2 ~nls +xz
    hdf5:
      variants: +fortran +hl +shared
    libfabric:
      variants: fabrics=sockets,tcp,udp,rxm
    libunwind:
      variants: +pic +xz
    openblas:
      variants: threads=openmp
    trilinos:
      variants: +amesos +amesos2 +anasazi +aztec +belos +boost +epetra +epetraext
        +ifpack +ifpack2 +intrepid +intrepid2 +isorropia +kokkos +ml +minitensor +muelu
        +nox +piro +phalanx +rol +rythmos +sacado +stk +shards +shylu +stokhos +stratimikos
        +teko +tempus +tpetra +trilinoscouplings +zoltan +zoltan2 +superlu-dist gotype=long_long
    xz:
      variants: +pic
    mesa:
      version: [21.3.8]
    mpi:
      require: mpich
    mpich:
      require: '@4.1.2 ~wrapperrpath ~hwloc'
    ncurses:
      require: '@6.3 +termlib'
    tbb:
      require: intel-tbb
    boost:
      version: [1.79.0]
      variants: +atomic +chrono +container +date_time +exception +filesystem +graph
        +iostreams +locale +log +math +mpi +multithreaded +program_options +random
        +regex +serialization +shared +signals +stacktrace +system +test +thread +timer
        cxxstd=17 visibility=global
    libffi:
      require: "@3.4.4"
    vtk-m:
      require: "+examples"
    cuda:
      version: [11.8.0]

  compilers:
  - compiler:
      spec: gcc@11.4.0
      paths:
        cc: /usr/bin/gcc
        cxx: /usr/bin/g++
        f77: /usr/bin/gfortran
        fc: /usr/bin/gfortran
      flags: {}
      operating_system: ubuntu20.04
      target: aarch64
      modules: []
      environment: {}
      extra_rpaths: []

  specs:
  - ecp-data-vis-sdk +adios2 +hdf5 +vtkm +zfp +paraview +cuda cuda_arch=75
```

</details>

<details><summary>Concretization</summary>

<pre>
 -   7pkzysi  embree@3.13.1%gcc@11.4.0~ipo+ispc build_system=cmake build_type=Release generator=make arch=linux-ubuntu20.04-aarch64
[+]  7hxi535      ^cmake@3.26.3%gcc@11.4.0~doc+ncurses+ownlibs build_system=generic build_type=Release arch=linux-ubuntu20.04-aarch64
[+]  ef47omc          ^curl@8.1.2%gcc@11.4.0~gssapi~ldap~libidn2~librtmp~libssh~libssh2+nghttp2 build_system=autotools libs=shared,static tls=openssl arch=linux-ubuntu20.04-aarch64
[+]  4jhk4yh              ^nghttp2@1.52.0%gcc@11.4.0 build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  vq4o7pj              ^openssl@3.1.2%gcc@11.4.0~docs+shared build_system=generic certs=mozilla arch=linux-ubuntu20.04-aarch64
[+]  wfmbgoq                  ^ca-certificates-mozilla@2023-05-30%gcc@11.4.0 build_system=generic arch=linux-ubuntu20.04-aarch64
[+]  iv32a2d              ^pkgconf@1.9.5%gcc@11.4.0 build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  fzluouk          ^ncurses@6.3%gcc@11.4.0~symlinks+termlib abi=none build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  kzszqob          ^zlib@1.2.13%gcc@11.4.0+optimize+pic+shared build_system=makefile arch=linux-ubuntu20.04-aarch64
[+]  whozcr3      ^gmake@4.4.1%gcc@11.4.0~guile build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  4pfsqtg          ^gnuconfig@2022-09-17%gcc@11.4.0 build_system=generic arch=linux-ubuntu20.04-aarch64
[+]  dou4tnz      ^intel-tbb@2021.9.0%gcc@11.4.0~ipo+shared+tm build_system=cmake build_type=Release cxxstd=default generator=make patches=12a9c3e arch=linux-ubuntu20.04-aarch64
[+]  yhk3kcy          ^hwloc@2.9.1%gcc@11.4.0~cairo~cuda~gl~libudev+libxml2~netloc~nvml~oneapi-level-zero~opencl+pci~rocm build_system=autotools libs=shared,static arch=linux-ubuntu20.04-aarch64
[+]  74eccrr              ^libpciaccess@0.17%gcc@11.4.0 build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  lunrww6                  ^libtool@2.4.7%gcc@11.4.0 build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  cybqwl7                  ^util-macros@1.19.3%gcc@11.4.0 build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  hqmyh2p              ^libxml2@2.10.3%gcc@11.4.0~python build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  dmdkgje                  ^libiconv@1.17%gcc@11.4.0 build_system=autotools libs=shared,static arch=linux-ubuntu20.04-aarch64
[+]  zxfypd4      ^ispc@1.20.0%gcc@11.4.0~ipo build_system=cmake build_type=Release generator=make arch=linux-ubuntu20.04-aarch64
[+]  fegwesy          ^bison@3.8.2%gcc@11.4.0 build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  esdggm2              ^diffutils@3.9%gcc@11.4.0 build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  b4ymrvu              ^gettext@0.21.1%gcc@11.4.0+bzip2+curses+git~libunistring+libxml2+tar+xz build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  h7rmakm                  ^tar@1.34%gcc@11.4.0 build_system=autotools zip=pigz arch=linux-ubuntu20.04-aarch64
[+]  zkgaasc                      ^pigz@2.7%gcc@11.4.0 build_system=makefile arch=linux-ubuntu20.04-aarch64
[+]  zojha6y              ^m4@1.4.19%gcc@11.4.0+sigsegv build_system=autotools patches=9dc5fbd,bfdffa7 arch=linux-ubuntu20.04-aarch64
[+]  mtu7bgv                  ^libsigsegv@2.14%gcc@11.4.0 build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  wr5hfa5              ^perl@5.38.0%gcc@11.4.0+cpanm+opcode+open+shared+threads build_system=generic arch=linux-ubuntu20.04-aarch64
[+]  gk4kscx                  ^berkeley-db@18.1.40%gcc@11.4.0+cxx~docs+stl build_system=autotools patches=26090f4,b231fcc arch=linux-ubuntu20.04-aarch64
[+]  v7cxzi5          ^flex@2.6.3%gcc@11.4.0+lex~nls build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  36ha6pi              ^findutils@4.9.0%gcc@11.4.0 build_system=autotools patches=440b954 arch=linux-ubuntu20.04-aarch64
[+]  fzgn7gd          ^llvm@14.0.6%gcc@11.4.0+clang~cuda~flang+gold~ipo+libomptarget~libomptarget_debug~link_llvm_dylib+lld+lldb+llvm_dylib~mlir+polly~python~split_dwarf~z3 build_system=cmake build_type=Release compiler-rt=runtime generator=ninja libcxx=runtime libunwind=runtime openmp=runtime patches=1f42874,25bc503,6379168,8248141,b216cff shlib_symbol_version=none targets=aarch64,all,arm version_suffix=none arch=linux-ubuntu20.04-aarch64
[+]  2aehw6p              ^binutils@2.40%gcc@11.4.0~gas+gold~gprofng+headers~interwork+ld+libiberty~lto~nls~pgo+plugins build_system=autotools compress_debug_sections=zlib libs=shared,static arch=linux-ubuntu20.04-aarch64
[+]  6zef44j                  ^texinfo@7.0.3%gcc@11.4.0 build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  iojwerc                  ^zstd@1.5.5%gcc@11.4.0+programs build_system=makefile compression=none libs=shared,static arch=linux-ubuntu20.04-aarch64
[+]  rcheb7w              ^libedit@3.1-20210216%gcc@11.4.0 build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  kgueuga              ^libffi@3.4.4%gcc@11.4.0 build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  jfm7wl6              ^ninja@1.11.1%gcc@11.4.0+re2c build_system=generic arch=linux-ubuntu20.04-aarch64
[+]  n7q2fwr                  ^re2c@2.2%gcc@11.4.0 build_system=generic arch=linux-ubuntu20.04-aarch64
[+]  uc7g5hr              ^perl-data-dumper@2.173%gcc@11.4.0 build_system=perl arch=linux-ubuntu20.04-aarch64
[+]  qcg7zcm          ^python@3.10.12%gcc@11.4.0+bz2+crypt+ctypes+dbm~debug+libxml2+lzma~nis~optimizations+pic+pyexpat+pythoncmd+readline+shared+sqlite3+ssl~tkinter+uuid+zlib build_system=generic patches=0d98e93,7d40923,f2fd060 arch=linux-ubuntu20.04-aarch64
[+]  ja4d3qc              ^bzip2@1.0.8%gcc@11.4.0~debug~pic+shared build_system=generic arch=linux-ubuntu20.04-aarch64
[+]  wmuhzov              ^expat@2.5.0%gcc@11.4.0+libbsd build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  ormxkmq                  ^libbsd@0.11.7%gcc@11.4.0 build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  sbf43az                      ^libmd@1.0.4%gcc@11.4.0 build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  jm67j6t              ^gdbm@1.23%gcc@11.4.0 build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  evoeirh              ^libxcrypt@4.4.35%gcc@11.4.0~obsolete_api build_system=autotools patches=4885da3 arch=linux-ubuntu20.04-aarch64
[+]  iptq6b7              ^readline@8.2%gcc@11.4.0 build_system=autotools patches=bbf97f1 arch=linux-ubuntu20.04-aarch64
[+]  loeyfnm              ^sqlite@3.42.0%gcc@11.4.0+column_metadata+dynamic_extensions+fts~functions+rtree build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  xqflduv              ^util-linux-uuid@2.38.1%gcc@11.4.0 build_system=autotools arch=linux-ubuntu20.04-aarch64
[+]  q7galae              ^xz@5.4.1%gcc@11.4.0+pic build_system=autotools libs=shared,static arch=linux-ubuntu20.04-aarch64
</pre>

</details>

Error:
```
$> spack -e . install /7pkzy
...
==> Installing embree-3.13.1-7pkzysii7qan7jz4xh6tihyirjxg7q4i [20/20]
==> No binary for embree-3.13.1-7pkzysii7qan7jz4xh6tihyirjxg7q4i found: installing from source
==> Using cached archive: /spack/var/spack/cache/_source-cache/archive/00/00dbd852f19ae2b95f5106dd055ca4b304486436ced0ccf842aec4e38a4df425.tar.gz
==> No patches needed for embree
==> embree: Executing phase: 'cmake'
==> embree: Executing phase: 'build'
==> Error: ProcessError: Command exited with status 2:
...
     304    cd /tmp/root/spack-stage/spack-stage-embree-3.13.1-7pkzysii7qan7jz4xh6tihyirjxg7q4i/spack-build-7pkzysi/kernels && /spack/opt/spack/__spack_path_placeholder__/__spack_path_placeholder__/_
            _spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholde/li
            nux-ubuntu20.04-aarch64/gcc-11.4.0/cmake-3.26.3-7hxi535v6dz7eu7z7flqd74za6rhma36/bin/cmake -D src=/tmp/root/spack-stage/spack-stage-embree-3.13.1-7pkzysii7qan7jz4xh6tihyirjxg7q4i/spack-sr
            c/kernels/bvh/bvh_refit.cpp -D dst=/tmp/root/spack-stage/spack-stage-embree-3.13.1-7pkzysii7qan7jz4xh6tihyirjxg7q4i/spack-build-7pkzysi/bvh/bvh_refit.cpp.avx512.cpp -P /tmp/root/spack-sta
            ge/spack-stage-embree-3.13.1-7pkzysii7qan7jz4xh6tihyirjxg7q4i/spack-src/common/cmake/create_isa_dummy_file.cmake
  >> 305    g++: error: unrecognized command-line option '-msse4.2'
     306    cd /tmp/root/spack-stage/spack-stage-embree-3.13.1-7pkzysii7qan7jz4xh6tihyirjxg7q4i/spack-build-7pkzysi/kernels && /spack/opt/spack/__spack_path_placeholder__/__spack_path_placeholder__/_
            _spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholde/li
            nux-ubuntu20.04-aarch64/gcc-11.4.0/cmake-3.26.3-7hxi535v6dz7eu7z7flqd74za6rhma36/bin/cmake -D src=/tmp/root/spack-stage/spack-stage-embree-3.13.1-7pkzysii7qan7jz4xh6tihyirjxg7q4i/spack-sr
            c/kernels/bvh/bvh_intersector_stream_filters.cpp -D dst=/tmp/root/spack-stage/spack-stage-embree-3.13.1-7pkzysii7qan7jz4xh6tihyirjxg7q4i/spack-build-7pkzysi/bvh/bvh_intersector_stream_fil
            ters.cpp.avx.cpp -P /tmp/root/spack-stage/spack-stage-embree-3.13.1-7pkzysii7qan7jz4xh6tihyirjxg7q4i/spack-src/common/cmake/create_isa_dummy_file.cmake
     307    [ 26%] Generating ../common/scene_curves.cpp.avx512.cpp
  >> 308    g++: error: unrecognized command-line option '-msse4.2'
...
  >> 488    make[1]: *** [CMakeFiles/Makefile2:493: kernels/CMakeFiles/embree_avx.dir/all] Error 2
     489    make[2]: Entering directory '/tmp/root/spack-stage/spack-stage-embree-3.13.1-7pkzysii7qan7jz4xh6tihyirjxg7q4i/spack-build-7pkzysi'
     490    [ 48%] Building CXX object kernels/CMakeFiles/embree_avx512.dir/__/geometry/curve_intersector_virtual_4i_mb.cpp.avx512.cpp.o
  >> 491    g++: error: unrecognized command-line option '-mavx512f'
  >> 492    g++: error: unrecognized command-line option '-mavx512dq'
  >> 493    g++: error: unrecognized command-line option '-mavx512f'
  >> 494    g++: error: unrecognized command-line option '-mavx512cd'
  >> 495    g++: error: unrecognized command-line option '-mavx512dq'
  >> 496    g++: error: unrecognized command-line option '-mavx512bw'
  >> 497    g++: error: unrecognized command-line option '-mavx512cd'
  >> 498    g++: error: unrecognized command-line option '-mavx512vl'
  >> 499    g++: error: unrecognized command-line option '-mf16c'
  >> 500    g++: error: unrecognized command-line option '-mavx2'
...
  >> 620    make[2]: *** [kernels/CMakeFiles/embree_avx512.dir/build.make:291: kernels/CMakeFiles/embree_avx512.dir/__/geometry/curve_intersector_virtual_8v.cpp.avx512.cpp.o] Error 1
  >> 621    g++: error: unrecognized command-line option '-mprefer-vector-width=256'; did you mean '-msve-vector-bits=256'?
  >> 622    make[2]: *** [kernels/CMakeFiles/embree_avx512.dir/build.make:305: kernels/CMakeFiles/embree_avx512.dir/__/geometry/curve_intersector_virtual_8i.cpp.avx512.cpp.o] Error 1
     623    make[2]: Leaving directory '/tmp/root/spack-stage/spack-stage-embree-3.13.1-7pkzysii7qan7jz4xh6tihyirjxg7q4i/spack-build-7pkzysi'
  >> 624    make[1]: *** [CMakeFiles/Makefile2:597: kernels/CMakeFiles/embree_avx512.dir/all] Error 2
     625    [ 51%] Linking CXX static library ../../liblexers.a
```

### Error message

<details><summary>Error message</summary>

<pre>
... see above
</pre></details>

### Information on your system

* **Spack:** 0.21.0.dev0 (a38535fa8ee17c6fcafeb7a4ab7665a27d2b569e)
* **Python:** 3.8.10
* **Platform:** linux-ubuntu20.04-neoverse_n1
* **Concretizer:** clingo

### Additional information

[spack-build-env.txt](https://github.com/spack/spack/files/12386581/spack-build-env.txt)
[spack-build-out.txt](https://github.com/spack/spack/files/12386582/spack-build-out.txt)

@aumuell

### General information

- [X] I have run `spack debug report` and reported the version of Spack/Python/Platform
- [X] I have run `spack maintainers <name-of-the-package>` and **@mentioned** any maintainers
- [X] I have uploaded the build log and environment files
- [X] I have searched the issues of this repo and believe this is not a duplicate

**Comment by aumuell** [comment]
There is already a PR, which addresses embree on ARM: https://github.com/spack/spack/pull/38083 - but I only tested that on macOS.

**Comment by eugeneswalker** [comment]
> There is already a PR, which addresses embree on ARM: #38083 - but I only tested that on macOS.

I just tested https://github.com/spack/spack/pull/38083 and the issue is not resolved for `embree@3.13.5` on ARM. 

`embree@3.13.5` is the latest version that can be used to build the `openvkl` according to current constraints applied when trying to concretize the following ECP DAV specs:
```
  - ecp-data-vis-sdk +adios2 +hdf5 +vtkm +zfp +paraview +cuda cuda_arch=75
  - ecp-data-vis-sdk +adios2 +hdf5 +vtkm +zfp +paraview +cuda cuda_arch=80
  - ecp-data-vis-sdk +adios2 +hdf5 +vtkm +zfp +paraview +cuda cuda_arch=90
```

```
==> Installing embree-3.13.5-2ctrs7b2te6k3ah7qu7rwimqk25gnaiy [20/20]
==> No binary for embree-3.13.5-2ctrs7b2te6k3ah7qu7rwimqk25gnaiy found: installing from source
==> Fetching https://github.com/embree/embree/archive/v3.13.5.tar.gz
==> No patches needed for embree
==> embree: Executing phase: 'cmake'
==> embree: Executing phase: 'build'
==> Error: ProcessError: Command exited with status 2:
    '/spack/opt/spack/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholde/linux-ubuntu20.04-aarch64/gcc-11.4.0/gmake-4.4.1-whozcr3nxucpd2q2lji2pejwk3vg6hvd/bin/make' '-j32'

31 errors found in build log:
     86     [ 15%] Building CXX object common/sys/CMakeFiles/sys.dir/condition.cpp.o
     87     cd /tmp/root/spack-stage/spack-stage-embree-3.13.5-2ctrs7b2te6k3ah7qu7rwimqk25gnaiy/spack-build-2ctrs7b/common/sys && /spack/lib/spack/env/gcc/g++ -DEMBREE_TARGET_SSE2 -DTASKING_TBB  -Wal
            l -Wformat -Wformat-security -Wno-class-memaccess -fno-strict-overflow -fno-delete-null-pointer-checks -fwrapv -fsigned-char -fPIE -fPIC -std=c++11 -fvisibility=hidden -fvisibility-inline
            s-hidden -fno-strict-aliasing -fno-tree-vectorize -D_FORTIFY_SOURCE=2  -DNDEBUG -O3  -msse2 -MD -MT common/sys/CMakeFiles/sys.dir/regression.cpp.o -MF CMakeFiles/sys.dir/regression.cpp.o.
            d -o CMakeFiles/sys.dir/regression.cpp.o -c /tmp/root/spack-stage/spack-stage-embree-3.13.5-2ctrs7b2te6k3ah7qu7rwimqk25gnaiy/spack-src/common/sys/regression.cpp
     88     [ 16%] Building CXX object common/sys/CMakeFiles/sys.dir/barrier.cpp.o
     89     cd /tmp/root/spack-stage/spack-stage-embree-3.13.5-2ctrs7b2te6k3ah7qu7rwimqk25gnaiy/spack-build-2ctrs7b/common/sys && /spack/lib/spack/env/gcc/g++ -DEMBREE_TARGET_SSE2 -DTASKING_TBB  -Wal
            l -Wformat -Wformat-security -Wno-class-memaccess -fno-strict-overflow -fno-delete-null-pointer-checks -fwrapv -fsigned-char -fPIE -fPIC -std=c++11 -fvisibility=hidden -fvisibility-inline
            s-hidden -fno-strict-aliasing -fno-tree-vectorize -D_FORTIFY_SOURCE=2  -DNDEBUG -O3  -msse2 -MD -MT common/sys/CMakeFiles/sys.dir/mutex.cpp.o -MF CMakeFiles/sys.dir/mutex.cpp.o.d -o CMake
            Files/sys.dir/mutex.cpp.o -c /tmp/root/spack-stage/spack-stage-embree-3.13.5-2ctrs7b2te6k3ah7qu7rwimqk25gnaiy/spack-src/common/sys/mutex.cpp
     90     cd /tmp/root/spack-stage/spack-stage-embree-3.13.5-2ctrs7b2te6k3ah7qu7rwimqk25gnaiy/spack-build-2ctrs7b/common/sys && /spack/lib/spack/env/gcc/g++ -DEMBREE_TARGET_SSE2 -DTASKING_TBB  -Wal
            l -Wformat -Wformat-security -Wno-class-memaccess -fno-strict-overflow -fno-delete-null-pointer-checks -fwrapv -fsigned-char -fPIE -fPIC -std=c++11 -fvisibility=hidden -fvisibility-inline
            s-hidden -fno-strict-aliasing -fno-tree-vectorize -D_FORTIFY_SOURCE=2  -DNDEBUG -O3  -msse2 -MD -MT common/sys/CMakeFiles/sys.dir/condition.cpp.o -MF CMakeFiles/sys.dir/condition.cpp.o.d
            -o CMakeFiles/sys.dir/condition.cpp.o -c /tmp/root/spack-stage/spack-stage-embree-3.13.5-2ctrs7b2te6k3ah7qu7rwimqk25gnaiy/spack-src/common/sys/condition.cpp
     91     cd /tmp/root/spack-stage/spack-stage-embree-3.13.5-2ctrs7b2te6k3ah7qu7rwimqk25gnaiy/spack-build-2ctrs7b/common/sys && /spack/lib/spack/env/gcc/g++ -DEMBREE_TARGET_SSE2 -DTASKING_TBB  -Wal
            l -Wformat -Wformat-security -Wno-class-memaccess -fno-strict-overflow -fno-delete-null-pointer-checks -fwrapv -fsigned-char -fPIE -fPIC -std=c++11 -fvisibility=hidden -fvisibility-inline
            s-hidden -fno-strict-aliasing -fno-tree-vectorize -D_FORTIFY_SOURCE=2  -DNDEBUG -O3  -msse2 -MD -MT common/sys/CMakeFiles/sys.dir/barrier.cpp.o -MF CMakeFiles/sys.dir/barrier.cpp.o.d -o C
            MakeFiles/sys.dir/barrier.cpp.o -c /tmp/root/spack-stage/spack-stage-embree-3.13.5-2ctrs7b2te6k3ah7qu7rwimqk25gnaiy/spack-src/common/sys/barrier.cpp
  >> 92     g++: error: unrecognized command-line option '-msse2'
  >> 93     g++: error: unrecognized command-line option '-msse2'
  >> 94     g++: error: unrecognized command-line option '-msse2'
  >> 95     make[2]: *** [common/math/CMakeFiles/math.dir/build.make:79: common/math/CMakeFiles/math.dir/constants.cpp.o] Error 1
     96     make[2]: Leaving directory '/tmp/root/spack-stage/spack-stage-embree-3.13.5-2ctrs7b2te6k3ah7qu7rwimqk25gnaiy/spack-build-2ctrs7b'
  >> 97     make[2]: *** [common/sys/CMakeFiles/sys.dir/build.make:79: common/sys/CMakeFiles/sys.dir/sysinfo.cpp.o] Error 1
  >> 98     make[2]: *** [common/simd/CMakeFiles/simd.dir/build.make:79: common/simd/CMakeFiles/simd.dir/sse.cpp.o] Error 1
     99     make[2]: *** Waiting for unfinished jobs....
     100    make[2]: Leaving directory '/tmp/root/spack-stage/spack-stage-embree-3.13.5-2ctrs7b2te6k3ah7qu7rwimqk25gnaiy/spack-build-2ctrs7b'
  >> 101    make[1]: *** [CMakeFiles/Makefile2:260: common/math/CMakeFiles/math.dir/all] Error 2
     102    make[1]: *** Waiting for unfinished jobs....
  >> 103    g++: error: unrecognized command-line option '-msse2'
  >> 104    make[1]: *** [CMakeFiles/Makefile2:286: common/simd/CMakeFiles/simd.dir/all] Error 2
  >> 105    g++: error: unrecognized command-line option '-msse2'
  >> 106    make[2]: *** [common/sys/CMakeFiles/sys.dir/build.make:93: common/sys/CMakeFiles/sys.dir/alloc.cpp.o] Error 1
  >> 107    g++: error: unrecognized command-line option '-msse2'
  >> 108    g++: error: unrecognized command-line option '-msse2'
  >> 109    make[2]: *** [common/sys/CMakeFiles/sys.dir/build.make:107: common/sys/CMakeFiles/sys.dir/filename.cpp.o] Error 1
  >> 110    make[2]: *** [common/sys/CMakeFiles/sys.dir/build.make:121: common/sys/CMakeFiles/sys.dir/library.cpp.o] Error 1
  >> 111    g++: error: unrecognized command-line option '-msse2'
  >> 112    make[2]: *** [common/sys/CMakeFiles/sys.dir/build.make:135: common/sys/CMakeFiles/sys.dir/thread.cpp.o] Error 1
  >> 113    g++: error: unrecognized command-line option '-msse2'
  >> 114    g++: error: unrecognized command-line option '-msse2'
  >> 115    make[2]: *** [common/sys/CMakeFiles/sys.dir/build.make:149: common/sys/CMakeFiles/sys.dir/string.cpp.o] Error 1
  >> 116    g++: error: unrecognized command-line option '-msse2'
  >> 117    make[2]: *** [common/tasking/CMakeFiles/tasking.dir/build.make:79: common/tasking/CMakeFiles/tasking.dir/taskschedulertbb.cpp.o] Error 1
     118    make[2]: Leaving directory '/tmp/root/spack-stage/spack-stage-embree-3.13.5-2ctrs7b2te6k3ah7qu7rwimqk25gnaiy/spack-build-2ctrs7b'
  >> 119    make[2]: *** [common/sys/CMakeFiles/sys.dir/build.make:163: common/sys/CMakeFiles/sys.dir/regression.cpp.o] Error 1
  >> 120    make[1]: *** [CMakeFiles/Makefile2:339: common/tasking/CMakeFiles/tasking.dir/all] Error 2
  >> 121    g++: error: unrecognized command-line option '-msse2'
  >> 122    make[2]: *** [common/sys/CMakeFiles/sys.dir/build.make:177: common/sys/CMakeFiles/sys.dir/mutex.cpp.o] Error 1
  >> 123    g++: error: unrecognized command-line option '-msse2'
  >> 124    make[2]: *** [common/sys/CMakeFiles/sys.dir/build.make:191: common/sys/CMakeFiles/sys.dir/condition.cpp.o] Error 1
  >> 125    make[2]: *** [common/sys/CMakeFiles/sys.dir/build.make:205: common/sys/CMakeFiles/sys.dir/barrier.cpp.o] Error 1
     126    make[2]: Leaving directory '/tmp/root/spack-stage/spack-stage-embree-3.13.5-2ctrs7b2te6k3ah7qu7rwimqk25gnaiy/spack-build-2ctrs7b'
  >> 127    make[1]: *** [CMakeFiles/Makefile2:234: common/sys/CMakeFiles/sys.dir/all] Error 2
     128    make[1]: Leaving directory '/tmp/root/spack-stage/spack-stage-embree-3.13.5-2ctrs7b2te6k3ah7qu7rwimqk25gnaiy/spack-build-2ctrs7b'
  >> 129    make: *** [Makefile:159: all] Error 2
```

CC @kwryankrattiger

**Comment by aumuell** [comment]
I don't think that there can be much done about embree@3.13.1 breaking on linux arm, but with #39749, 3.13.5 should work. Can this now be closed?
